examples/portable/executor_runner/open_file_by_fd.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Probably not necessary to put this file to repo. Maybe just mention it in commit message?